### PR TITLE
Format small

### PR DESCRIPTION
**Pull request by uribgp**
Related to #1496

Description: breaking up large pull for f-strings

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

File: tests/ignite/contrib/handlers/test_mlflow_logger.py [hunk 1]
```diff
@@ -282,7 +282,7 @@ def dummy_handler(engine, logger, event_name):
             global_step = engine.state.get_event_attrib_value(event_name)
             v = global_step * 0.1
             true_values.append(v)
-            logger.log_metrics({"{}".format("test_value"): v}, step=global_step)
+            logger.log_metrics({"'test_value'": v}, step=global_step)
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_mlflow_logger.py:285]:
```suggestion
            logger.log_metrics({"test_value": v}, step=global_step)
```

File: tests/ignite/contrib/handlers/test_mlflow_logger.py [hunk 2]
```diff
@@ -240,7 +240,7 @@ def dummy_handler(engine, logger, event_name):
         global_step = engine.state.get_event_attrib_value(event_name)
         v = global_step * 0.1
         true_values.append(v)
-        logger.log_metrics({"{}".format("test_value"): v}, step=global_step)
+        logger.log_metrics({"'test_value'": v}, step=global_step)
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_mlflow_logger.py:243]: Same here as below

File: tests/ignite/contrib/handlers/test_polyaxon_logger.py [hunk 3]
```diff
@@ -244,7 +244,7 @@ def update_fn(engine, batch):
 
     def dummy_handler(engine, logger, event_name):
         global_step = engine.state.get_event_attrib_value(event_name)
-        logger.log_metrics(step=global_step, **{"{}".format("test_value"): global_step})
+        logger.log_metrics(step=global_step, **{"'test_value'": global_step})
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_polyaxon_logger.py:247]: Same here

File: tests/ignite/contrib/handlers/test_polyaxon_logger.py [hunk 4]
```diff
@@ -268,7 +268,7 @@ def update_fn(engine, batch):
 
         def dummy_handler(engine, logger, event_name):
             global_step = engine.state.get_event_attrib_value(event_name)
-            logger.log_metrics(step=global_step, **{"{}".format("test_value"): global_step})
+            logger.log_metrics(step=global_step, **{"'test_value'": global_step})
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_polyaxon_logger.py:271]: Same here

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @uribgp ! Hope this is the last chunk :)
I left few comments to fix a small issue.

**Comment by uribgp** [comment]
OK I removed those.  Unfortunately there's still quite a few files left... somewhere around 50.

**Review by vfdev-5 — APPROVED** [review]
Thanks @uribgp !

**Pull request merged**